ports/atmel-samd/boards/sparkfun_lumidrive/mpconfigboard.h

```diff
@@ -0,0 +1,51 @@
+#define MICROPY_HW_BOARD_NAME "SparkFun LUMIDrive"
+#define MICROPY_HW_MCU_NAME "samd21g18"
+
+//#define MICROPY_HW_NEOPIXEL (&pin_PA06)
+
+// Clock rates are off: Salae reads 12MHz which is the limit even though we set it to the safer 8MHz.
+#define SPI_FLASH_BAUDRATE  (8000000)
+
+#define SPI_FLASH_MOSI_PIN   &PIN_PA08
+#define SPI_FLASH_MISO_PIN   &PIN_PA14
+#define SPI_FLASH_SCK_PIN    &PIN_PA09
+#define SPI_FLASH_CS_PIN     &PIN_PA13
+
+// All port(pin) definitions should be deleted unless PA06 -- SERCOM0/PAD[2]
+// Will need to scour the repo to figure out why this was done. 
+// I'm guessing these ports are defined elsewehere by default rather than needing to
+// repeatedly define them. Saves time to name outliers than participants in
+// some cases. 
```

Comment:
These are definitely poorly named. They ensure some pins are not automatically reset. We're moving to a different way of doing it so if your board works as expected then this should be fine.

Reply:
End of day here for me, didn't mean to resolve without comment. Sorry about the extraneous comments. I removed them in latest commit.